src/uniprotkb/components/entry/ComputationallyMappedSequences.tsx

```diff
@@ -15,15 +15,20 @@ import apiUrls from '../../../shared/config/apiUrls';
 import { Location, LocationToPath } from '../../../app/config/urls';
 
 import { MessageLevel } from '../../../messages/types/messagesTypes';
+import { Sequence } from '../../../shared/types/sequence';
+import { OrganismData } from '../../adapters/namesAndTaxonomyConverter';
 
-// NOTE: Jie told me this would be replaced by a different
-// format he called "FASTA" entry
 type ProteinEntryLight = {
-  accession: string;
+  id: string;
+  sequence: Sequence;
   entryType: string;
-  sequenceLength: number;
+  uniProtkbId: string;
+  proteinName: string;
+  organism: OrganismData;
   geneName: string;
-  geneNameType: string;
+  proteinExistence: string;
```

Comment:
This feels like it could be a union of possible values instead of the generic `string`

Reply:
Same as https://github.com/ebi-uniprot/uniprot-website/blob/master/src/tools/blast/types/apiSequenceData.ts#L23 - it is free text (e.g. "1: Evidence at protein level")

Reply:
Looks like there's only 5 possible strings here https://www.uniprot.org/help/protein_existence

Reply:
You're correct, it's not free text - but I remember why this was added as a string, it's because the format doesn't match what is displayed on the current website (no numbers) and so it was planned to strip the number at some point. Added an enum for now and will make a minor jira to remember to update the string when displaying it